sofria_mediaId.js

```diff
@@ -21,16 +24,20 @@ const fqPath = xx
 
 const content = fse.readFileSync(fqPath).toString();
 const suffix = fqPath.split(".").reverse()[0];
-// replace xxx with language from command input; similarly, replace yyy with version 
+// replace xxx with first chars 0-2 of USX FILESET DIRECTORY and yyy with chars 3-5
```

Comment:
parsing the fileset to determine the language and version is not valid in all cases